tests/shutdown/shutdown.pm

```diff
@@ -25,7 +25,8 @@ use power_action_utils 'power_action';
 use utils;
 
 sub run {
-    power_action('poweroff');
+    my $keep = check_var('BACKEND', 's390x');
```

Comment:
I would propose that we reopen the bug and add soft-failure here, or at least add a comment explaining this workaround. WDYT?

Reply:
I'm not sure about re-opening the bug, as it was related to flatpak, and  I'm not sure this one is,. I'll need to investigate on this. Though yes, I think a workaround would be better.

Reply:
we can also file a new one, as previous problem was visible on zkvm and was fixed there.

Reply:
That is, if we consider this as a bug at all. The strange thing is that it happens only on s390

Reply:
I reported this https://bugzilla.suse.com/show_bug.cgi?id=1177446 and that https://bugzilla.suse.com/show_bug.cgi?id=1177445, let's see what feedback we'll get.

Reply:
Good job here! So, let's add soft-failure to this workaround and merge it.

Reply:
I also removed another workaround. I will follow-up on this and remove this current one as well once bug is fixed. Updated VRs indescription, waiting for zVM to pass then we can remove WIP.